src/parsers/js/transformers/babel6/codeExample.txt

```diff
@@ -1,5 +1,8 @@
-export default function ({types: t}) {
+export default function (babel) {
+  let { types: t } = babel;
```

Comment:
Should we use `const` instead of `let` here as it's unlikely the user is going to change any property of `babel`?